sensor_msgs/PointCloud input
---
sensor_msgs/PointCloud output
